ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -0,0 +1,532 @@
+
+Once the configuration has been deployed, an automated validation of the designed configuration versus the running configuration can be executed using the included `validation` playbook, to ensure that:
+
+- all links are operational and cabled as desired
+- BGP adjacencies are operational
+- and more...
+
+This quickly identifies any issues that could otherwise be overlooked and a redundant link could sit non-operational for months unless cumbersome manual testing is executed again per-device throughout the fabric.
+
+## Day 2 Operations
+The term “Day 2 Operations” describes most, if not all network configuration changes that occur after the initial day 1 configuration.
+
+The really time consuming day-to-day tasks such as:
+- adding/removing new leaf switches
+- adding/removing new VLANs
+- creating/modifying/deleting port profiles
+- configuring/modifying ports towards end systems
+
+no longer have to be performed on a device-by-device basis across the entire fabric.
+
+A configuration change can easily be applied to the Ansible YAML files and the scope for a given change can be specified, e.g. should the new VLAN 100 be added to all or a subset of leaf switches.
+
+Once your changes are complete, 
+you re-deploy the configuration changes to the relevant network devices, again either via CloudVision Portal or directly via EOS eAPI.
+
+### Day 2 Operations Example
+
+Let's imagine that you want to create a new tenant in your existing fabric.
+This would require at least the following configuration:
+- VRF(s)
+- VLAN(s)
+- SVI(s)
+
+You could create a YAML file, in this example called DC1_TENANTS_NETWORKS.yml, describing the fabric-wide generic configuration for the tenant:
+
+```
+# DC1 Tenants Networks
+# Documentation of Tenant specific information - Vlans/VRFs
+tenants:
+  # Tenant A Specific Information - VRFs / VLANs
+  Tenant_A:
+    mac_vrf_vni_base: 10000
+    vrfs:
+      Tenant_A_OP_Zone:
+        vrf_vni: 10
+        vtep_diagnostic:
+          loopback: 100
+          loopback_ip_range: 10.255.1.0/24
+        svis:
+          110:
+            name: Tenant_A_OP_Zone_1
+            tags: [opzone]
+            enabled: true
+            ip_address_virtual: 10.1.10.1/24
+          111:
+            vni_override: 50111
+            name: Tenant_A_OP_Zone_2
+            tags: [opzone]
+            enabled: true
+            ip_address_virtual: 10.1.11.1/24
+          112:
+            vni_override: 50112
+            name: Tenant_A_OP_Zone_3
+            tags: [opzone]
+            enabled: true
+            ip_address_virtual: 10.1.12.254/24
+```
+
+You would then describe the scope for where this new tenant should exist in the inventory.yml file, as described in earlier chapters.
+For sake of simplicity, let's say you defined just one pair of leaf switches that would serve this tenant.
+
+After running the relevant playbook, their tenant-related configuration would end up looking like this:
+```
```

Comment:
[md031](https://github.com/DavidAnson/markdownlint/blob/v0.24.0/doc/Rules.md#md031) error

Reply:
md040 error.

This should be: ```eos